examples/dummy_honest/examples/dummy_honest.rs

```diff
@@ -0,0 +1,217 @@
+use aleph_bft::{run_session, Recipient};
+use aleph_bft_mock::{
+    Data, DataProvider, FinalizationHandler, Hasher64, Keychain, Loader, PartialMultisignature,
+    Saver, Signature, Spawner,
+};
+use chrono::Local;
+use clap::Parser;
+use codec::{Decode, Encode};
+use futures::{
+    channel::{mpsc, oneshot},
+    FutureExt, StreamExt,
+};
+use libp2p::{
+    development_transport,
+    floodsub::{self, Floodsub, FloodsubEvent},
+    identity,
+    mdns::{Mdns, MdnsConfig, MdnsEvent},
+    swarm::{NetworkBehaviourEventProcess, SwarmBuilder},
+    NetworkBehaviour, PeerId, Swarm,
+};
+use log::{debug, info, warn};
+use parking_lot::Mutex;
+use std::{error::Error, io::Write, sync::Arc, time::Duration};
+
+const ALEPH_PROTOCOL_NAME: &str = "aleph";
+
+/// Dummy honest node example.
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    /// Our index
+    #[clap(long)]
+    my_id: usize,
+
+    /// Size of the committee
+    #[clap(long)]
+    n_members: usize,
+
+    /// Number of data to be finalized
+    #[clap(long)]
+    n_finalized: usize,
+}
+
+#[tokio::main]
+async fn main() {
+    env_logger::builder()
+        .format(|buf, record| {
+            writeln!(
+                buf,
+                "{} {}: {}",
+                record.level(),
+                Local::now().format("%Y-%m-%d %H:%M:%S%.3f"),
+                record.args()
+            )
+        })
+        .filter(None, log::LevelFilter::Debug)
+        .init();
+
+    let args = Args::parse();
+
+    info!(target: "dummy-honest", "Getting network up.");
+    let (network, mut manager) = Network::new().await.unwrap();
+    let (close_network, exit) = oneshot::channel();
+    tokio::spawn(async move { manager.run(exit).await });
+
+    let data_provider = DataProvider::new();
+    let (finalization_handler, mut finalized_rx) = FinalizationHandler::new();
+
+    let units = Arc::new(Mutex::new(vec![]));
```

Comment:
Comment for here and an analogous place in one of the other files: calling this `units` (and the things below `unit_loader` and `unit_saver`) suggests that it is a vector of units and per my understanding it is not -- it holds serialized data, units for now, but will likely also hold other things in the future.  I would call it either `buffer` or `raw_bytes` or something like that.

Reply:
Right now this is called the same in `LocalIO`. For now those names are accurate (`unit_loader` and `unit_saver`). Do you think we should change it there too? In future we would also like to save/load Alerts. Maybe something like `backup_loader`, `backup_saver`?

Reply:
The names of the arguments in `aleph_bft::LocalIO` should be indeed  `backup_loader`, `backup_saver` or `backup_reader/writer`. The user doesn't have to know that we will be storing units there (and actually should not assume that, because we likely will store other things there as well). On the other hand, the internal structs name `UnitSaver` and `UnitLoader` can be kept, because this is accurate for now.

Reply:
Ok, `aleph_bft::LocalIO::new` arguments can be changed in https://github.com/Cardinal-Cryptography/AlephBFT/pull/197. Here I would go with changing those variable names to `backup`, `backup_loader` and `backup_saver`.

Reply:
renamed, also removed `units` as `Loader` and `Saver` should not (?) share data